srcCpp/RTIDDSImpl.cxx

```diff
@@ -2829,6 +2811,25 @@ IMessagingReader *RTIDDSImpl<T>::CreateReader(
     }
 }
 
+std::string getQosProfile(const char *topicName)
+{
+    std::string qosProfile;
```

Comment:
set it to "", some compilers might no initialize this to an empty string.

Reply:
I check this and seems like it not needed. But I already changed just in case. :+1: 

From http://www.cplusplus.com/reference/string/string/string/
Since C++98
(1) empty string constructor (default constructor)
    Constructs an empty string, with a length of zero characters.